spimple/utils.py

```diff
@@ -89,8 +89,16 @@ def Gaussian2D(xin, yin, GaussPar=(1., 1., 0.), normalise=True):
     S0, S1, PA = GaussPar
     Smaj = np.maximum(S0, S1)
     Smin = np.minimum(S0, S1)
-    A = np.array([[1. / Smin ** 2, 0],
+
+    # hackaround for runtime warning
+    if 0 in (Smaj, Smin):
```

Comment:
This seems very hacky. Why is it necessary? I'll try to clean up the logic but I need to understand when this happens

Reply:
Note to self - this will happen if there are null sub-bands, these should be dropped from the fit